exopy/__main__.py

```diff
@@ -202,6 +203,8 @@ def main(cmd_line_args=None):
     parser.add_argument("-r", "--reset-app-folder",
                         help='Reset the application startup folder.',
                         action='store_true')
+    parser.add_argument("-x", "--execute",
```

Comment:
Since this is specific to the measurement plugin-workspace, I think a more specific argument `--measurement-execute` with no short form could be better. I assume you do not plan to use this directly through the command line but more through an external app.

Reply:
Seems reasonable, I'll do it